local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Thu Dec 17 19:39:23 PST 2020
+sdk.dir=/Users/owenlarosa/Library/Android/sdk
```

Comment:
This shouldn't be in git. Whoever downloads this code will get an error that your folder doesn't exist on their laptop.